assets/scss/_custom.scss

```diff
@@ -1352,3 +1352,40 @@ div.alert > em.javascript-required {
   outline: none;
   padding: .5em 0 .5em 0;
 }
+
+/* CSS for 'figure' full-screen display */
+ 
+/* Define styles for full-screen overlay */
+.figure-fullscreen-overlay {
+  position: fixed;
+  inset: 0;
+  z-index: 9999;
+  background-color: rgba(255, 255, 255, 0.95); /* White background with some transparency */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  box-sizing: border-box;
+}
+
+/* CSS class to scale the image when zoomed */
+.figure-zoomed {
+  transform: scale(1.2);
+}
+
+/* Define styles for full-screen image */
+.figure-fullscreen-img {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: contain; /* Maintain aspect ratio and fit within the container */
+}
+
+/* Define styles for close button */
+.figure-close-button {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  cursor: pointer;
+  font-size: 60px;
+  color: #333;
```

Comment:
(nit)
Could use CSS `calc()` for responsive sizing here.

Reply:
Attempted to incorporate this feedback comment in the latest push.